Automattic/wp-calypso
# Stats: Integrate module settings API to the highlights settings panel for period toggling

Related to #77425 

## Proposed Changes

* Apply modules settings fetch and update API that is tackled from D112693-code.

## Testing Instructions



* <strike>Sandbox API with D112693-code</strike> D112693-code is live now.
* Spin this change with the local site.
* Navigate to Stats > `Traffic` page.
* Select the `30-day highlights` option to update the highlights data.
* Reload the page after the update API is completed.
* Ensure the `30-day highlights` option is selected as remembered.

## Pre-merge Checklist
